### PR TITLE
[Issue Templates] Create issue templates for Modular CLI and Dev Console

### PR DESCRIPTION
**Pull request by goldiegadde**
- Create issue templates for Modular CLI and Dev Console
- auto assign the appropriate labels at the time of issue creation
- Update the mojo bug report and FR templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/dev_console_issue.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Develper Console Issue report
+description: Create a issue report to help us improve the Developer Console.
```
Comment (dualvtable) [inline .github/ISSUE_TEMPLATE/dev_console_issue.yaml:2]: > Create a issue report

Missing "an" -> "Create an issue to help us..."

File: .github/ISSUE_TEMPLATE/mojo_bug_report.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-name: Beta Bug report
-description: Create a bug report for the beta program to help us improve Mojo
-title: "[BETA]: "
+name: Mojo bug report
```
Comment (dualvtable) [inline .github/ISSUE_TEMPLATE/mojo_bug_report.yaml:1]: Should we be consistent with "bug report" and "Issue Report"? Not a big deal - but good to change if you can.

File: .github/ISSUE_TEMPLATE/mojo_bug_report.yaml [hunk 3]
```diff
@@ -35,9 +36,9 @@ body:
     id: Context
     attributes:
       label: System information
-      description: Create a new notebook in Mojo Playground, execute the following code, and paste the output back into this text box.
+      description: What version of Mojo are you using?
       value: |
-        - What OS did you do install modular CLI/mojo on ?
+        - What OS did you do install mojo on ?
```
Comment (dualvtable) [inline .github/ISSUE_TEMPLATE/mojo_bug_report.yaml:41]: "Mojo" capitalization

File: .github/ISSUE_TEMPLATE/mojo_bug_report.yaml [hunk 4]
```diff
@@ -25,8 +25,9 @@ body:
       label: Steps to reproduce
       description: Provide the specific steps to reproduce the issue. 
       value: |
-        - Include relevant code snippet that did not work as expected.
+        - Include relevant code snippet or link to code that did not work as expected.
         - If applicable, add screenshots to help explain the problem.
+        - If using playground, name the pre-existing notebook that failed and the steps that led to failure.
```
Comment (dualvtable) [inline .github/ISSUE_TEMPLATE/mojo_bug_report.yaml:30]: "If using the Playground, ..."

**Review by dualvtable — APPROVED** [review]
Left some minor comments, but lgtm otherwise. Thanks!

File: .github/ISSUE_TEMPLATE/mojo_bug_report.yaml (hunk 2 above)
Comment (goldiegadde) [reply .github/ISSUE_TEMPLATE/mojo_bug_report.yaml:1]: I deliberately made them distinct as for Mojo we have both bug and enhancement templates.

**Pull request merged**